lib/messagebird.js

```diff
@@ -226,6 +226,69 @@ module.exports = function (accessKey, timeout) {
       }
     },
 
+    mms: {
+
+      /**
+       * Get a mms message
+       *
+       * @param {String} id
+       * @param {Function} callback
+       * @return {void}
+       */
+      read: function (id, callback) {
+        httpRequest('GET', '/mms/' + id, callback);
+      },
+
+      /**
+       * Send a mms message
+       *
+       * @param {Object} params
+       * @param {Function} callback
+       * @return {void}
+       */
+      create: function (params, callback) {
+        if (params.recipients instanceof Array) {
+          params.recipients = params.recipients.join(',');
+        }
+        httpRequest('POST', '/mms', params, callback);
+      },
+
+      /**
+       * Lists mms messages. Pagination is optional. If a limit is set, an
+       * offset is also required.
+       *
+       * @param {Number} limit
+       * @param {Number} offset
+       * @param {Function} callback
+       * @return void
+       */
+      list: function (limit, offset, callback) {
+        var params = null;
```

Comment:
Overwriting this later makes the code hard to read. Maybe you could create the params object all the time and leaving the values out that are missing?

Reply:
This actually exactly how this is done throughout this entire project. I am not a big fan of handling it like that either but IMO for our first change we should try to keep consistent with existing codebase and any changes to the current convention or solution approach as separate tickets and first discuss them with birds with longer feathers 😉.

Reply:
Good point but I tried to use the same coding style with the codebase purposely.